roles/foreman_postgresql/defaults/main.yml

```diff
@@ -1,4 +1,4 @@
 ---
 foreman_postgresql_database: foreman
 foreman_postgresql_user: foreman
-foreman_postgresql_password: foreman
+foreman_postgresql_password: "{{ lookup('password', '/tmp/passwordfile chars=ascii_letters,digits,hexdigits,punctuation') }}"
```

Comment:
Why we store it in /tmp? If we want to persist this, we should store it somewhere else, e.g. $pwd/.postgresql_password

Given reports we've seem with traditional installer, perhaps we should use vault from the beginning? I don't insist, just interested in your longer term thoughts.